playbooks/roles/clean_puppet/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I'd drop this role. It was originally added in the foreman-bats because you want to start with a clean install, but in the ansible context it doesn't really make sense (unless you do it before the foreman-installer run).